src/main/java/org/broadinstitute/hellbender/engine/Walker.java

```diff
@@ -0,0 +1,27 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.exceptions.GATKException;
+
+/**
+ * An abstract class to represent built-in walkers that inherit directly from {@link GATKTool}.
+ * Created by jonn on 6/28/18.
+ */
+public abstract class Walker extends GATKTool {
+
+    @Override
+    final protected ReferenceDataSource getReferenceDataSource() {
+        throw new GATKException("Should never access ReferenceDataSource in child classes of AssemblyRegionWalker.");
```

Comment:
`AssemblyRegionWalker` should be `Walker` - here and below!

Reply:
Good catch!

Fixed!